EasyPost.Tests/ServicesTests/CustomsInfoServiceTest.cs

```diff
@@ -31,6 +32,30 @@ public async Task TestCreate()
             Assert.Equal("NOEEI 30.37(a)", customsInfo.EelPfc);
         }
 
+        [Fact]
+        [CrudOperations.Create]
+        [Testing.Parameters]
+        public async Task TestCreateWithCustomsItems()
+        {
+            UseVCR("create_with_customs_items");
+
+            Dictionary<string, object> data = Fixtures.BasicCustomsInfo;
+            data["customs_items"] = new List<Dictionary<string, object>>
+            {
+                Fixtures.BasicCustomsItem,
+            };
+
+            CustomsInfo customsInfo = await Client.CustomsInfo.Create(data);
+
+            Assert.IsType<CustomsInfo>(customsInfo);
+            Assert.StartsWith("cstinfo_", customsInfo.Id);
+            Assert.NotEmpty(customsInfo.CustomsItems);
+            foreach (CustomsItem item in customsInfo.CustomsItems)
+            {
+                Assert.StartsWith("cstitem_", item.Id);
+            }
```

Comment:
Can we instead just append this assertion to the already existing test? The fixture already has a single item included in the customs info data.

Reply:
Oh, it does? I didn't see that, sounds good.